content/en/continuous_integration/troubleshooting.md

```diff
@@ -84,15 +84,18 @@ If you cannot see the tests wall time it is likely that the CI provider metadata
 
 ### The tests wall time is not what is expected
 
+#### How wall time is calculated
 The wall time is defined as the time difference between the start time of the first test and the end time of the last test for the given pipeline.
 
 This is done using the following algorithm:
 
 1. Compute a hash based on CI information to group the tests.
-  a. If the tests have `ci.job.url`, this information is used to calculate the hash.
-  b. If the tests don’t have `ci.job.url`, the hash is calculated using `ci.pipeline.id` + `ci.pipeline.name` + `ci.pipeline.number`.
-2. Calculate wall time is associated with every given hash. **Note**: There can be multiple wall times for a certain pipeline, for example, if there are multiple jobs that execute tests.
-3. If there are multiple wall times, the maximum wall time calculated for a given pipeline in a certain commit is used.
```

Comment:
I _think_ this was wrong, at least from my experience. I think the right explanation is in https://github.com/DataDog/dd-trace-rb/issues/1837#issuecomment-1016403185